drupal/values.yaml

```diff
@@ -155,6 +155,7 @@ php:
         drush updatedb -y
         if [ -f $DRUPAL_CONFIG_PATH/core.extension.yml ]; then
           drush config-import -y
+          drush cc drush
```

Comment:
@kirkkala this makes sense, but could we add a short comment to explain why this is needed, especially since this deployment step is not commonly used.

Reply:
Added in 6060033, sorry not too scientific though. And I guess I couldn't have add comments in between the command. Or did you mean to `printf` the comment?